server/http.go

```diff
@@ -134,6 +134,9 @@ func (p *Plugin) initializeRouter() {
 	// Firehose webhook setup for channel subscriptions
 	instanceRouter.HandleFunc(makeAPIRoute(routeAPISubscribeWebhook), p.handleResponseWithCallbackInstance(p.httpSubscribeWebhook)).Methods(http.MethodPost)
 
+	// To support Plugin v2.x webhook URL
+	apiRouter.HandleFunc(routeAPISubscribeWebhook, p.handleResponseWithCallbackInstance(p.httpSubscribeWebhook)).Methods(http.MethodPost)
```

Comment:
There are two types of webhook URLs that have the shape `/webhook` and `api/v2/webhook`. It looks like we already support the `/webhook` path with no instance ID, but not one that does contain an instance ID:

https://github.com/mattermost/mattermost-plugin-jira/blob/d1e233af901f5a4c3b652fee260857c74d1c6d44/server/http.go#L86C22-L86C42

You can run `/jira webhook` and see the section called "Legacy webhooks". That feature is essentially the original webhook implementation we had, that allows you to post to Mattermost without having to create subscriptions. Customers still use that when they don't want Mattermost to have access to their Jira server. So we still need to make sure that's covered in the `instanceRouter`

Reply:
sure @mickmister looking into this.

Reply:
@mickmister added additional endpoint